src/Resources/views/storefront/page/search/layout/leftFilterLayout.html.twig

```diff
@@ -10,17 +10,18 @@ is configured on the left side.
     {% set productColumns = 'col-lg-8 col-xl-9' %}
 {% endif %}
 
+{# This value is explicitely checked for versions <6.4.4.0, instead of using the sidebar value #}
+{% if block is not defined %}
+    {% set block = { sectionPosition: 'sidebar' } %}
+{% else %}
+    {% set block = block|merge({ sectionPosition: 'sidebar' }) %}
+{% endif %}
+
 <div class="row">
-    <div class="cms-element-sidebar-filter {{ filterColumns }}">
-        <div id="filter-panel-wrapper"
-             class="filter-panel-wrapper"
-             data-offcanvas-filter-content="true">
-            {% sw_include '@Storefront/storefront/component/listing/filter-panel.html.twig' with {
-                listing: page.listing,
-                sidebar: true
-            } %}
-        </div>
-    </div>
+    {% sw_include '@Storefront/storefront/element/cms-element-sidebar-filter.html.twig' with {
+        listing: page.listing,
+        sidebar: 1
```

Comment:
Use `true` like Shopware

Reply:
Seems like SW is not really sure themselves, but changed it to boolean on our side

![image](https://user-images.githubusercontent.com/51320851/162385922-9946bd97-87f1-4c48-9659-35b46ceaca2e.png)